packages/react-components/src/organisms/__tests__/TeamCard.test.tsx

```diff
@@ -48,3 +49,18 @@ it('pluralises when more than one team member', () => {
   );
   expect(getByText('3 Team Members')).toBeVisible();
 });
+
+it('renders a lab count for multiple labs', () => {
+  const { getByText } = render(<TeamCard {...teamCardProps} labCount={13} />);
+  expect(getByText('13 Labs')).toBeVisible();
+});
+
+it('renders a lab count for a single lab using singular form', () => {
+  const { getByText } = render(<TeamCard {...teamCardProps} labCount={1} />);
+  expect(getByText('1 Lab')).toBeVisible();
```

Comment:
is this the right way to test singular forms? I wonder if this test wouldn't pass even if plural was used?

Reply:
I believe that's covered by these tests 
https://github.com/yldio/asap-hub/blob/215fe56c8e8eed1fa5235fe1011fb77b934cb654/packages/react-components/src/utils/__tests__/text.test.tsx#L9-L25